DEVELOPMENT.md

```diff
@@ -0,0 +1,19 @@
+# Dev Tooling
+
+## Saving a new sample from a development environment
+
+In order to make changes to the sample scenes within the SDK, you need to create a new Unity project and
+import the samples into your project. We provide a script that allows you to save your completed changes
+back into the SDK.
```

Comment:
I'm not sure what "saving back into the SDK" means

Reply:
Fixed